model_utils/tests/models.py

```diff
@@ -4,25 +4,24 @@
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 
+from model_utils import Choices
+from model_utils.fields import SplitField, MonitorField, StatusField
+from model_utils.managers import QueryManager, InheritanceManager
 from model_utils.models import (
     SoftDeletableModel,
     StatusModel,
     TimeFramedModel,
     TimeStampedModel,
 )
-from model_utils.tracker import FieldTracker, ModelTracker
-from model_utils.managers import QueryManager, InheritanceManager
-from model_utils.fields import SplitField, MonitorField, StatusField
 from model_utils.tests.fields import MutableField
-from model_utils import Choices
-
+from model_utils.tests.managers import CustomSoftDeleteManager
+from model_utils.tracker import FieldTracker, ModelTracker
 
 
 class InheritanceManagerTestRelated(models.Model):
     pass
 
 
-
```

Comment:
Indeed, that was @carljm preference. I respected the PEP8 when I refactored the tests though. Only because people are more used to PEP8 than anything else.